src/vector/v.maxent.swd/v.maxent.swd.html

```diff
@@ -1,54 +1,79 @@
+<em>v.maxent.swd</em> Produce a set of SWD files as input to MaxEnt 
+3.3.3k+, based on a point vector layer(s) of species distribution(s), a 
+point layer with background points and user-defined raster layers. The 
+user can also let the function create the background point file. 
+(respecting region and mask, if set).
+
+<p>
+The SWD file format is a simple CSV-like file format as described 
+in Elith et al. 2011. The first three fields provide the species name, 
+x-coordinate and y-coordinate, while subsequent fields contain the 
+values of the user-selected environmental parameters. Note that the 
+coordinates are not used by the MaxEnt program if all environmental 
+data is given in the SWD format file.
+
+<p>
+The user can provide one or more species point files, which will be 
+combined in one SWD file. The map names of both the species point 
+layers and the environmental parameters can be replaced by alias names, 
+which will be used by MaxEnt.
+
+<p>
+If alias names are used, a CSV file (alias_file) can be created with 
+alias names in the first column and map names in the second column, 
+separated by comma, without header. This can be used as input in 
+r.maxent.lambdas (but note that this addon does not seem to work with 
+outputs from Maxent 3.4+).
 
 <h2>NOTES</h2>
 
 The SWD output files are simple comma delimited text
 files (.csv) and can therefore be easily read in in for example R
 and subsequently used in other models / functions.
 
+<p>
+This addon is a vector based alternative to <em>r.out.maxent_swd</em>. It can
+be more efficient with sparse data points. The main difference is that with
+this addon you can have more than one sample point per raster cell. 
+points per raster cell. If this is not desired, use r.out.maxent_swd. 
+Another difference is the option to export the predictor raster layers 
+to a user-defined folder. This can be used in Maxent, Maxnet addon for 
+R or other software.
+
 <h2>SEE ALSO</h2>
 
 <ul>
-    <li>MaxEnt 3.3.3k (version 3.4.1 untested, but should work as well)
-    (<a href="http://biodiversityinformatics.amnh.org/open_source/maxent/">
-    http://biodiversityinformatics.amnh.org/open_source/maxent/</a>)</li>
-    <li>The <a
-    href="http://svn.osgeo.org/grass/grass-addons/grass6/raster/r.out.maxent_swd/r.out.maxent_swd.html">
-    r.out.maxent_swd</a> script (for GRASS GIS 6.*) for an alternative
-    implementation</li>
+	<li>MaxEnt 3.4.1 (<a 
+	href="http://biodiversityinformatics.amnh.org/open_source/maxent/"> 
+	http://biodiversityinformatics.amnh.org/open_source/maxent/</a>)</li> 
+	<li><a 
+	href="https://grass.osgeo.org/grass-stable/manuals/addons/r.out.maxent_swd.html"> 
+	r.out.maxent_swd</a>, the original implemention using species 
+	distribution data in raster format.</li>
+	<li><a 
+	href="https://grass.osgeo.org/grass-stable/manuals/addons/r.maxent.lambdas.html"> 
+	r.maxent.lambdas</a> addon to computes raw or logistic prediction 
+	maps from MaxEnt lambdas files.</li>
+	
```

Comment:
```suggestion
	<li><a href="r.out.maxent_swd.html">r.out.maxent_swd</a>, the original
	implementation using species distribution data in raster format.</li>
	<li><a href="r.maxent.lambdas.html">r.maxent.lambdas</a> addon to
	compute raw or logistic prediction maps from MaxEnt lambdas files.</li>
```

typo fixed + changed URLs
(absolute URLs only used for core modules while addon manual page URLs are relative)